### ISSUE TITLE
pivot table cells without values should be visually differentiated from those with values

### ISSUE THREAD

**Issue by kdoh** [metabase/metabase#1601]
large pivot tables with a non uniform distribution of values can be hard to parse. Adding a visual differentiator between cells with values those those without values should help with this.

**Comment by mazameli** [comment]
Accidentally stumbled on this old issue. Do we think this would still be relevant if we lose vertical dividers?
